thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiFragment.java

```diff
@@ -151,6 +151,18 @@ public View onCreateView(final LayoutInflater inflater, @Nullable final ViewGrou
     @Override
     public void onDestroy() {
         super.onDestroy();
+        //FIXME handle attach/detach state
+        TiLog.v(TAG, "isChangingConfigurations = " + getActivity().isChangingConfigurations());
```

Comment:
Please move all the `Log` method into it's own method like `logFragmentInformation` or something like that...